Makefile

```diff
@@ -23,9 +24,14 @@ all: build
 test: ## Run tests.
 	@go test -count=1 $(PKGS)
 
+
+test-e2e: ## Run e2e integration tests
```

Comment:
I am a huge fan of testing and integration test, and think it's great work, but I am wondering if this is worth the extra effort of building and running the binary, when you could test the `listBundlesMain(...)` function without any extra toil/preparation? I don't see what we are gaining from running integration tests here vs. regular unit tests of the `listBundlesMain(...)` function

Reply:
I think we should have separate unit tests along with integration tests. The idea of this integration test(and future ones) would be to test things from the outside(like how our users would use it) and make sure that the interface we give out to our clients is proper.(Stuff like proper formatting of output, how we take in input through STDIN etc). While I agree that this feature right now is too trivial, the idea still stands.

Reply:
I am sold :) , thank you